src/Router/ModuleFileRouter.php

```diff
@@ -45,6 +45,8 @@ class ModuleFileRouter implements RequestHandlerInterface
     protected $subdir;
 
     protected $contenttype;
+
+    protected string $moduledir;
```

Comment:
PHP Parse error:  syntax error, unexpected 'string' (T_STRING), expecting function (T_FUNCTION) or const (T_CONST) in /var/www/loris/src/Router/ModuleFileRouter.php on line 49